docs/keepers.md

```diff
@@ -0,0 +1,51 @@
+# Keepers
+
+## Overview
+
+## Common Conventions
+
+The Keeper class follows similar conventions as the smart contracts. See [smart_contracts.md](./smart_contracts.md) for explanations.
+
+## Classes
+### Keeper
+
+The base interface for all other keeper classes. `generate_actions_for_timestep` returns an array of `action` objects that is run by [experiment.py](../experiments/experiment.py)
+
+### VaultKeeper
+
+Class representation of a keeper who opens vaults. `open_max_vaults` will open the maximum possible number of vaults for each ilk.
+
+### NaiveVaultKeeper
+
+The Naive Vault Keeper will open the maximum number of vaults possible at each timestep.
+
+### AuctionKeeper
+
+The base interface for auction keeper classes. 
+`find_bids_to_place` will take `now` (current timestep as String TODO) as input and will output a dictionary with `ilk_ids` as keys and lists of bids as values.
```

Comment:
`now` is an int, it's passed in from the loop in the `run` function of an experiment

Reply:
Should I explicitly state types of all maker variables the first time we mention them, or just let the reader look at the MCD docs?

Reply:
Mmm might as well state the type, it'll be useful for us as well